ztp/resource-generator/Containerfile

```diff
@@ -15,6 +16,11 @@ WORKDIR $PGT_ROOT
 RUN make build
 WORKDIR $SC_ROOT
 RUN make build
+# Build pgt2acm
+WORKDIR $PGT2ACM_ROOT
+RUN go mod vendor
+RUN make build
```

Comment:
Does it make sense to encapsulate all the logic to build ('go mod vendor' in this case) inside the corresponding Makefile so 'make build' drives the build entirely? 

I have typically seen that components which claim to perform vendoring and are dockerized use a 'vendor' directory which is versioned and as part of the corresponding Dockerfile, a simple COPY of that directory is performed before building. Can we assess this approach ?

/cc @fontivan

Reply:
Fixed. Added a commit for it.